client/pluginmanager/csimanager/manager.go

```diff
@@ -39,7 +39,7 @@ func New(config *Config) Manager {
 	}
 
 	return &csiManager{
-		logger:    config.Logger,
+		logger:    config.Logger.Named("csiManager"),
```

Comment:
For some reason we usually snake-case the logger names (probably because it's closer to the file names in a lot of places? I dunno).

```suggestion
		logger:    config.Logger.Named("csi_manager"),
```